components/conversation.js

```diff
@@ -25,6 +25,10 @@ const SCREEN_PLAYING = ScreenOutConfig.ScreenMode.PLAYING;
 const SCREEN_OFF = ScreenOutConfig.ScreenMode.OFF;
 const DEFAULT_SAMPLE_RATE_IN = 16000;
 const DEFAULT_SAMPLE_RATE_OUT = 24000;
+// The max number of bytes of audio that can be sent in a single chunk.
+// Not documented anywhere by google, but sending more than this amount
+// throws an INVALID_REQUEST error stating that audio_in is too long.
+const MAX_FRAME_LENGTH = 31 * 1024; 
```

Comment:
Was this just figured out by process of elimination? Love me the prime number :D

Reply:
Sadly, yes. I'm very unfamiliar with the gRPC stuff so if you know a better way, let me know. It's susceptible to changes on their end which is obviously bad.